pyaedt/emit_core/results/revision.py

```diff
@@ -648,7 +655,7 @@ def protection_level_classification(
                         else:
                             filtering = True
 
-                        if instance.get_value(mode_power) > max_power and filtering:
+                        if instance.has_valid_values() and instance.get_value(mode_power) > max_power and filtering:
```

Comment:
@jsalant22 Are there some reasons for not having valid values that could indicate an interference problem (e.g. saturation)? If so, should we be treating those as one of the problematic classifications? Right now, if I'm following the code correctly, the max_power value will be -200 for any case where `instance.has_valid_values()=false` and it will be treated like there is no interference issue.